learn/features/treasury.md

```diff
@@ -1,79 +1,45 @@
 ---
 title: Treasury
-description: As a Polkadot parachain, Moonbeam has an on-chain treasury controlled by council members, enabling stakeholders to submit proposals to further the network.
+description: Moonbeam has an on-chain Treasury controlled by Treasury Council members, enabling stakeholders to submit proposals to further the network.
 ---
 
 # Treasury on Moonbeam
 
 ## Introduction {: #introduction }
 
-A Treasury is an on-chain managed collection of funds. Each Moonbeam-based network has a community Treasury for supporting network initiatives to further the network, funded by a percentage of transaction fees of the network it is on and managed by the Council.
+The Moonbeam Treasury is an on-chain collection of funds launched at the network's genesis. The Treasury was pre-funded with 0.5% of the total token supply at network launch and accumulates GLMR steadily as {{ networks.moonbeam.inflation.parachain_bond_Treasury }}% of the parachain bond reserve inflation goes to the Treasury. The Moonbeam Treasury is intended to help fund key projects that help maintain and further the growth of the Moonbeam network.
```

Comment:
the variable here is not rendering correctly `{{ networks.moonbeam.inflation.parachain_bond_Treasury }}`.

You have `{{ networks.moonbeam.inflation.parachain_bond_inflation }}` or `{{ networks.moonbeam.inflation.parachain_bond_treasury }}` But I think you need to be clear that 80% of the yearly 1.5% of inflation goes to treasury